packages/hardhat-core/sample-projects/typescript/scripts/deploy.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "hardhat";
```

Comment:
Consider adding the same comment header that appears in `sample-projects/javascript/scripts/deploy.ts`.

Reply:
I didn't add the explanation because this is mandatory when you use typescript, which is expected by any ts user.